FSharp-4.5/FS-1053-span.md

```diff
@@ -498,7 +511,7 @@ machinery to express read-only references.
 
 # Notes
 
-* Implicit address-of is not applied for parameters to let-bound functions.  For example:
+* Implicit address-of was intially implemented for member methods but was taken out due to scoping rationale. Currently it is **not implemented**. If it was done again, it wouldn't be applied for parameters to let-bound functions. For example:
```

Comment:
@TIHan Could you reword this a bit and place under `Alternatives` to capture the discussion we just had, e.g. that this is part of the corresponding C# design, and the example where the scope of the F# tmp is narrow, and that we will seek to re-instate this in F# 5.0 especially if API ergonomics make it necessary? thanks

Reply:
Yup will do.